_content/av/dhammapada-verses_yuttadhammo.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Dhammapada Verses"
+authors:
+  - yuttadhammo
+subcat: poetry
```

Comment:
This isn't poetry itself, it's a `course`

See `av/an-course_bodhi` for an example on how to do this kind of entry